script.js

```diff
@@ -47,4 +50,119 @@ buttonElement.addEventListener('click', function (event) {
     subElement.classList.toggle('active');
 });
 
+buttonRight.addEventListener('click', function (){
+    move('right');
+}); 
+buttonLeft.addEventListener('click', function (){
+    move('left');
+});
+
+let images = document.querySelectorAll('.carousel object');
+let slider = [];
+for (let i = 0; i < images.length; i++) {
+    slider[i] = images[i].data;
+    images[i].remove();
+}
+let boxes = document.querySelectorAll('.carousel__box');
+for (let i = 0; i < boxes.length; i++) {
+    boxes[i].remove();
+}
+
+let carouselTextObject = document.createElement('div');
+carouselTextObject = carouselText;
+
+let step = 0;
+let carousel =  Array.from(document.querySelectorAll('.carousel__box'));
 
+function carouselObject (position, left) {
+    let objectDiv = document.createElement('div');
+    let objectImg = document.createElement('object');
```

Comment:
Check out `document.createDocumentFragment` maybe that is better to be used?

Reply:
Here I need elements, because I assign them style and  further. But I can change document.createElement to document.createDocumentFragment in another place, where I create only div for text:
let carouselTextObject = document.createDocumentFragment();